.github/workflows/test.yml

```diff
@@ -0,0 +1,37 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node-version: [10.x, 16.x]
```

Comment:
Maybe make sense to test on 12 and 14 too, sometimes regressions happen

Reply:
done